### PR TITLE
New Rule: no-class-bindings

### LINKED ISSUE #1762: Add new rule to deprecate passing `classBinding` and `classNameBindings` to component invocations
Deprecate passing `classBinding` and `classNameBindings` as arguments.

References: 

* https://github.com/emberjs/rfcs/blob/master/text/0691-deprecate-class-binding-and-class-name-bindings.md

### PR DESCRIPTION
**Pull request by scalvert**
Fixes #1762

### CODE REVIEW

File: docs/rule/no-class-bindings.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# no-class-bindings
+
+TODO: context about the problem goes here
+
+TODO: what the rule does goes here
```
Comment (rwjblue) [inline docs/rule/no-class-bindings.md:5]: Needs to be fleshed out

File: docs/rule/no-class-bindings.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+```hbs
+{{!-- TODO: Example 2  --}}
+```
+
+## Migration
+
+TODO: suggest any fast/automated techniques for fixing violations in a large codebase
+
+* TODO: suggestion on how to fix violations using find-and-replace / regexp
+* TODO: suggestion on how to fix violations using a codemod
```
Comment (rwjblue) [inline docs/rule/no-class-bindings.md:34]: 2️⃣ 💵

File: docs/rule/no-class-bindings.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Configuration
+
+TODO: exclude this section if the rule has no extra configuration
```
Comment (rwjblue) [inline docs/rule/no-class-bindings.md:38]: 🥈

File: docs/rule/no-class-bindings.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+## Related Rules
+
+* [TODO-related-rule-name1](related-rule-name1.md)
+* [TODO-related-rule-name2](related-rule-name2.md)
+
+## References
+
+* TODO: link to relevant documentation goes here
+* TODO: link to relevant function spec goes here
+* TODO: link to relevant guide goes here
```
Comment (rwjblue) [inline docs/rule/no-class-bindings.md:53]: 2️⃣ 💩

**Review by rwjblue — CHANGES_REQUESTED** [review]

**Comment by rwjblue** [comment]
Can you target this against `master` instead of `next`?

**Comment by scalvert** [comment]
Docs, pah! Who needs 'em. :)

P.S. Will get right on it.

**Comment by scalvert** [comment]
I can target master, but I'll need to cherry-pick this off next.

**Pull request closed without merging**